### PR TITLE
Styling/cc struture and theme

### PR DESCRIPTION
**Pull request by carlymclift**
#### What's this PR do?
* Fixes the pop-up boxes to render correctly
* Styles the story edit view to live render what the user types 

#### Where should the reviewer start?
Story Edit View files 

#### How should this be manually tested?
Open localhost:3000

#### Any background context you want to provide?
NOT been tested yet

#### Screenshots (if appropriate)

LiveDisplay

Questions:
Do we need a more readable font for the writing part or should it stay as more of a script font?

### CODE REVIEW

**Review by nickhartdev — APPROVED** [review]
I'm loving the theme! The cursive font is readable, to me at least. We might want to get some other inputs on that, though.

**Comment by GreysonElkins** [comment]
I think titles can stay script, but the body should maybe change to something a little more legible.
Love the ink pen <3

File: src/components/Bookshelf/Book.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ const Book = ({ story, onClick, popup }) => {
   return (
     <>
       <div 
-        data-tip data-for='global'
+        data-tip={prompt}
```
Comment (GreysonElkins) [inline src/components/Bookshelf/Book.js:27]: what's going on here?

**Review by GreysonElkins — APPROVED** [review]
Looks great!

File: src/components/Bookshelf/Book.js (hunk 1 above)
Comment (carlymclift) [reply src/components/Bookshelf/Book.js:27]: This fixes the issue with the pop-up trying to render everything at once

Comment (Abdeboskey) [reply src/components/Bookshelf/Book.js:27]: I really wanna know more about the tool-tip package!

File: src/components/StoryEditView/LiveDisplay.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+const LiveDisplay = ({props}) => {
+    console.log("c", props)
+    return (
+      <div id="script-writing-sec">
+          <h2>{props.title}</h2>
+          <p>{props.story}</p>
+      </div>
+    );
+  }
+
```
Comment (Abdeboskey) [inline src/components/StoryEditView/LiveDisplay.js:12]: This is such a clever and really cool feature!!!

File: src/components/StoryEditView/StoryEditView.css [hunk 3]
```diff
@@ -1,8 +1,35 @@
...
+  flex-flow: column;
   text-align: center;
-  width: 80%;
+  width: 40%;
   max-width: 650px;
   margin: auto;
-}
\ No newline at end of file
+}
+
```
Comment (Abdeboskey) [inline src/components/StoryEditView/StoryEditView.css]: I think we gotta move the background that is currently in App.js over to this component

File: src/components/App/App.css [hunk 4]
```diff
@@ -1,38 +1,6 @@
...
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
+body {
+  background-image: url(../../assets/Story_read.jpg);
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
```
Comment (Abdeboskey) [inline src/components/App/App.css]: This rule should be moved to story edit view I believe!

**Review by Abdeboskey — APPROVED** [review]
This feature is AWESOME. there is so much going on that makes this application really fun to use.

**Pull request merged**